navigating-many-to-one.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Managing many-to-one and one-to-many database relations'
+description: "Addressing the n+1 problem and navigating one-to-many relationships with Xata's new approach."
+image:
+  src: 'https://raw.githubusercontent.com/xataio/mdx-blog/main/images/csv-import-export-cover.jpg'
+  alt: 'Many-to-one relationship'
+author: Alejandro Martínez Vieites
+date: 09-01-2023
+published: true
+slug: navigating-many-to-one
+---
+
+As our [launch week](/blog/launch-week-august-2023) comes to a close, we're looking at APIs and their intricate relationships. Today, we're introducing a new syntax that simplifies navigating these relationships. While Xata already provides a great way to retrieve related data using [link columns](/docs/concepts/data-model#link) for the _many-to-one_ relationship, we're addressing the challenge of navigating the reverse [relationship](https://xata.io/docs/concepts/data-model#links-and-relations) (_one-to-many_). This new approach improves efficiency and helps avoid the _n + 1 problem_.
+
+Let's consider a basic schema. Within the schema, there's a _posts_ table containing certain information and establishing a connection to the corresponding author. This author-related data belongs in the _users_ table.
+
+![](image/many-to-one-relationships.png)
+
+When querying the _posts_ table we are able to retrieve the title, likes, and the author's name (and any other fields present in the _users_ table):
```

Comment:
If we don't plan to show SDK calls for these, we can introduce this section with something like:

> To explain the N+1 problem, we're going to show what API requests are sent of the network. This is what the Xata SDKs are doing behind the scenes.

Reply:
If we can show the SDK calls similar to other blogs, I really think we should. @SferaDev @philkra I'm pretty sure we have SDK support, how hard would it be to match some of these examples?

Reply:
In this case, because we're showing the responses as well and because we're explaining the problem of round-trips, showing only the API calls might be better.

However, how to call the reverse links via the SDK somewhere in the blog would be really good.

Reply:
TS & Python SDK snippets are in